app-backend/db/src/main/scala/com/azavea/rf/database/TemplateDao.scala

```diff
@@ -0,0 +1,79 @@
+package com.azavea.rf.database
+
+import java.sql.Timestamp
+
+import com.azavea.rf.database.Implicits._
+import com.azavea.rf.datamodel._
+import doobie._
+import doobie.implicits._
+import doobie.postgres._
+import doobie.postgres.implicits._
+import cats._
+import cats.data._
+import cats.effect.IO
+import cats.implicits._
+import java.util.UUID
+
+import scala.concurrent.Future
```

Comment:
I don't _think_ we should need `Future` in any of the `Dao`s -- sorry if this is included because you copy-pasted from something I worked on that included a `Future` import